### PR TITLE
VetTec: Frontend logic for when user answers if completed 1990 Form#16450

### PR DESCRIPTION
**Pull request by zurbergram**
## Description

As a Veteran who has not completed a 1990 yet but want to apply for VetTec I need to be taken to the 1990 interview after I complete my 0994 interview.

## Testing done
- [x] local testing
- [x] unit testing

## Screenshots
screen shot 2019-01-28 at 2 52 56 pm

## Acceptance criteria
- [x] If the user answers 'Yes' to 'Have you already submitted an application for VA Education Benefits (VA Form 22-1990)?' then we will not send them to the 1990 interview after completing/submitting the 0994 interview; the user will only see the confirmation page for 0994 and then they are finished
- [x] If the user answers 'No' to 'Have you already submitted an application for VA Education Benefits (VA Form 22-1990)?' then we will show the user the confirmation page for 0994 with a button that says 'Continue to VA Form 22-1990)

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by devin-mccurdy — APPROVED** [review]
LGTM

**Comment by zurbergram** [comment]
@department-of-veterans-affairs/frontend-review-group Could I get a code review on this at your earliest convenience. Thank you!

File: src/applications/edu-benefits/0994/containers/ConfirmationPage.jsx [hunk 1]
```diff
@@ -27,6 +33,9 @@ class ConfirmationPage extends React.Component {
   }
 
   componentDidMount() {
+    if (!_.get(this.props.form.data, 'appliedForVAEducationBenefits', true)) {
+      window.location.replace(url1990);
```
Comment (rianfowler) [inline src/applications/edu-benefits/0994/containers/ConfirmationPage.jsx]: Will the veteran be able to see the confirmation information with this behavior?

**Review by rianfowler — CHANGES_REQUESTED** [review]

File: src/applications/edu-benefits/0994/containers/ConfirmationPage.jsx (hunk 1 above)
Comment (zurbergram) [reply src/applications/edu-benefits/0994/containers/ConfirmationPage.jsx]: Not if they answer the question on the page with "Have you already submitted an Application for VA Education Benefits (VA Form 22-1990)?" with the answer "No". 

I asked @Jennywalter if this behavior was ok and she said yes. 

The story is https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/16450

**Review by rianfowler — APPROVED** [review]

File: src/applications/edu-benefits/0994/containers/ConfirmationPage.jsx (hunk 1 above)
Comment (zurbergram) [reply src/applications/edu-benefits/0994/containers/ConfirmationPage.jsx]: Changing this behavior, adding a button to Confirmation page to direct to start of 1990

**Pull request merged**